prow/gerrit/adapter/adapter.go

```diff
@@ -509,6 +537,11 @@ func (c *Controller) processChange(logger logrus.FieldLogger, instance string, c
 				break
 			}
 		}
+
+		// Suspect that inrepoconfig takes long time, add a log for its duration
```

Comment:
Nit: Better to say "We suspect that inrepoconfig takes a long time, so add a log for its duration." Or if you want to avoid "we", you can do "Inrepoconfig probably takes a long time, so add a log for its duration to measure and confirm." As it stands, "suspect that inrepconfig takes a long time" reads as "you, reader, suspect that inrepoconfig takes a long time".

Reply:
doen